Resources/Prototypes/_Goobstation/Balloons/package.yml

```
@@ -0,0 +1,85 @@
+    - id: BoxDarts
+      amount: 1
+    - id: MonkeyCube
+      prob: 0.7
+    - id: ClothingOuterSuitMonkey   # Extra
+      amount: 1
+      prob: 0.4
+    - id: BoxDarts
+      amount: 1
+      prob: 0.4
+    - id: MonkeyCube
+      prob: 0.4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)